src/app/scheduling/datetimeeditor/datetimeeditor-sample-1/datetimeeditor-sample-1.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="sample-wrapper">
```

Comment:
Simplify the structure, removing all but the following

```suggestion
<div class="datetime-wrapper">
    <igx-input-group>
        <input type="text" igxInput [igxDateTimeEditor]="'dd/MM/yyyy'" [(ngModel)]="date"/>
    </igx-input-group>
</div>

```

Reply:
Keeping the `<p>{{ date }}</p>` because I was told to show how the [(ngModel)] binded data was in the input was updated in another place.